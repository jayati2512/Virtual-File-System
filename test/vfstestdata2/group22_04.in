createvfs createvfs
createvfs demo 2
mountvfs demo
makedir /home/ b
makedir /home/ c
makedir /home/ d
removefile
listdir /home 3 /tmp/a.txt 
unmountvfs demo
