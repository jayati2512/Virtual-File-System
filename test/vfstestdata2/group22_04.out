VFS_INSUFFICIENT_ARGUMENTS
createvfs_SUCCESS
mountvfs_SUCCESS
makedir_SUCCESS
FILE_SYSTEM_FULL
FILE_SYSTEM_FULL
VFS_INSUFFICIENT_ARGUMENTS
INVALID_FLAG
unmountvfs SUCCESS
